docs/includes/fundamentals/as-avs/AtlasSearchTest.php

```diff
@@ -127,7 +130,7 @@ public function autocompleteSearchTest(): void
      * @runInSeparateProcess
      * @preserveGlobalState disabled
      */
-    public function vectorSearchTest(): void
+    public function testVectorSearch(): void
```

Comment:
This test is still returning an empty result set, I don't get why.

Reply:
Maybe this is because the titles of the sample data aren't related to the plot field values? I can investigate.

Reply:
The issue was in the model class that was used.